be/src/exprs/condition_expr.cpp

```diff
@@ -436,18 +448,22 @@ class VectorizedCoalesceExpr : public Expr {
         return builder.build(ColumnHelper::is_all_const(columns));
     }
 
-    StatusOr<ColumnPtr> _evaluate_complex(const Columns& columns) { // without only-null columns
-        int size = columns[0]->size();
+    StatusOr<ColumnPtr> _evaluate_complex(const Columns& inputs) { // without only-null columns
```

Comment:
there is no need to deal const_column for coalesceexpr, they will use other branch.

Reply:
Got it